File: templates/python/fr/text/code/cornerBeepers.py
# Fichier: Corner Jetons .py
# -----------------------------
# Place un jeton dans chaque coin
from karel.stanfordkarel import *
def main():
   # répéter le corps 4 fois
   for i in range(4):
	   laisser_jeton()
	   avancer()
	   avancer()
	   avancer()
	   tourner_gauche()

File: templates/java/fr/text/code/cornerBeepers.java
/*
 * Fichier: CoinJetons.java
 * ------------------------
 * Place un jeton dans chaque coin
 */
import stanford.karel.*;
public class CoinJetons extends Karel {
   public void run() {
      /* répéter le corps 4 fois */
      for(int i = 0; i < 4; i++) {
      	laisserJeton();
      	avancer();
      	avancer();
      	avancer();
      	tournerGauche();
      }
   } 
}